addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -87,6 +95,126 @@ var addArticleToSection = function (sectionName, pageName, backend) {
 }];
 };
 
+const fillInFileInput = async (input, name, type, content) => {
+    const blob = await fetch(`data:${type};base64,${content}`).then(res => res.blob());
```

Comment:
why do we need to fetch it ? (can't we just base 64 decode it ?)

Reply:
Actually awa-odoo asked me the same question. Although not the straight way to do it, it seems to be the simplest one because following some search on internet, efficient implementation are quite long (convert base64 to blob). Using that tricks is concise and still efficient and it is confined to the tests. I found that solution on stackoverflow. What do you think ?  (note: converted to use only await as suggested below)

Reply:
yes that's ok so :) it's in a test anyway